packages/component-base/src/browser-utils.js

```diff
@@ -4,14 +4,23 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 
-export const isAndroid = /android/i.test(navigator.userAgent);
+const testUserAgent = (regexp) => regexp.test(navigator.userAgent);
```

Comment:
We don't really need `/android/i` because it turns out to always be [capitalized](https://developers.whatismybrowser.com/useragents/explore/operating_system_name/android/).